src/Plugin/Commerce/PaymentGateway/OpenInvoice.php

```diff
@@ -0,0 +1,194 @@
+ *   },
+ * )
+ */
+class OpenInvoice extends OffsitePaymentGatewayBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return [
+      'merchant_account' => '',
+      'client_user' => '',
+      'client_password' => '',
+      'skin_code' => '',
+      'hmac' => '',
+      'shopper_locale' => '',
+      'recurring' => '',
+      'state' => '',
+      'use_checkout_form' => '',
+    ] + parent::defaultConfiguration();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $form = parent::buildConfigurationForm($form, $form_state);
+    $form['merchant_account'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Merchant Account'),
+      '#default_value' => $this->configuration['merchant_account'],
+      '#required' => TRUE,
+    ];
+
+    $form['client_user'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Client User'),
+      '#default_value' => $this->configuration['client_user'],
+      '#required' => TRUE,
+    ];
+
+    $form['client_password'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Client Password'),
+      '#default_value' => $this->configuration['client_password'],
+      '#required' => TRUE,
+    ];
+
+    $form['skin_code'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Skin Code'),
+      '#default_value' => $this->configuration['skin_code'],
+      '#required' => TRUE,
+    ];
+
+    $form['hmac'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('HMAC Key'),
+      '#default_value' => $this->configuration['hmac'],
+      '#required' => TRUE,
+    ];
+
+    $form['shopper_locale'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Shopper locale'),
+      '#default_value' => $this->configuration['shopper_locale'],
+      // @link https://docs.adyen.com/developers/hpp-manual#createaskin
+      '#options' => array_map('t', [
+        'zh' => 'Chinese – Traditional',
+        'cz' => 'Czech',
+        'da' => 'Danish',
+        'nl' => 'Dutch',
+        'en_GB' => 'English – British',
+        'en_CA' => 'English – Canadian',
+        'en_US' => 'English – US',
+        'fi' => 'Finnish',
+        'fr' => 'French',
+        'fr_BE' => 'French – Belgian',
+        'fr_CA' => 'French – Canadian',
+        'fr_CH' => 'French – Swiss',
+        'fy_NL' => 'Frisian',
+        'de' => 'German',
+        'el' => 'Greek',
+        'hu' => 'Hungarian',
+        'it' => 'Italian',
+        'li' => 'Lithuanian',
+        'no' => 'Norwegian',
+        'pl' => 'Polish',
+        'pt' => 'Portuguese',
+        'ru' => 'Russian',
+        'sk' => 'Slovak',
+        'es' => 'Spanish',
+        'sv' => 'Swedish',
+        'th' => 'Thai',
+        'tr' => 'Turkish',
+        'uk' => 'Ukrainian',
+      ]),
+      '#required' => TRUE,
+    ];
+
+    $form['recurring'] = [
+      '#type' => 'select',
+      '#title' => t('Recurring contract'),
```

Comment:
this `t()` should be used consistently

Reply:
It is because currently extended forms not let $this->t('...') or I am not sure about what is the problem in this case

Reply:
Use string translation trait